docs/trouble_shooting.md

```diff
@@ -61,4 +61,23 @@ This guide (based on minikube and others should be similar) explains general inf
     ✗ Preparing nodes 📦
     ERRO[22:52:13] docker run error: exit status 125
     Error: failed to create cluster: docker run error: exit status 125
-   ```
\ No newline at end of file
+   ```
+
+## Can provision cluster but calico-node-xx pod keeps CrashLoopBackOff status
+
+   Check the pod CIDR and serivce CIDR you specified in `cluster.yaml` have no overlap with provisioned node CIDR, and each data center you specified in `machines.yaml` has different node CIDR setting, for complete node CIDR settings in all data centers, please refer to: https://control.softlayer.com/network/subnets
```

Comment:
Start at the beginning

Reply:
I think one thing we are not clear by now is how the private IP get allocated..
refer to https://control.softlayer.com/network/subnets and you will see a set of subnets and which one will be selected seems under the hood of IBM cloud or we are not clear yet

that's a TODO for us and  I think we should have some info about this, otherwise, it's still unclear to customer/user how/why there is a conflict because he has no idea where's the private IP comes from

Reply:
Done, see: https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/pull/191/commits/fa3792b8c73ed05ac7e05b4574471da821467244